tests/checkpoint/test_checkpoint.py

```diff
@@ -1,1962 +0,0 @@
-from __future__ import annotations
-
-import logging
```

Comment:
Reviewers, please take a look at these tests and make sure they're okay to delete. I'm primarily focused on keeping E2E/integration tests alive (and those are covered in other files)